### DISCUSSION TITLE
Highlight causing errors while generating angular application build

### DISCUSSION THREAD

**Discussion by rhtmhr895** [highlight/highlight#9583] [Q&A]
Hey I am using highlight for session tracking in my angular application, the integration was working fine till last week, after which I am constantly getting errors whenever I am trying to generate a build for my application, the error is about missing modules graphql, graphql-request, rrweb and opentelemetry although most of them get resolved when i install the modules but a specific one still presists

Error: node_modules/highlight.run/dist/client/src/graph/generated/operations.d.ts:2:10 - error TS2305: Module '"graphql-request/build/cjs/types"' has no exported member 'GraphQLClientRequestHeaders'.

2 import { GraphQLClientRequestHeaders } from 'graphql-request/build/cjs/types';

On further debugging when i checked the node module folder for graphql-request the folder structure is as below graphql-request/dist not as build and neither any export for this GraphQLClientRequestHeaders is available and now I am stuck on this issue and have no solution to it can anyone guide me how can I handle this one.

My angular version is 17^, highlight version is 9.5^ and graphql is the latest one i.e. 7.1

**Comment by Vadman97** [comment]
Hi @rhtmhr895 do you know what twas the last version of highlight.run that was installed that did work?

**Reply by rhtmhr895** [reply]
@Vadman97  it was  ^9.1.1
